docs/reference/mpremote.rst

```diff
@@ -428,8 +428,9 @@ Shortcuts can be defined using the macro system.  Built-in shortcuts are:
 - ``cat``, ``edit``, ``ls``, ``cp``, ``rm``, ``mkdir``, ``rmdir``, ``touch``: Aliases for ``fs <sub-command>``
 
 Additional shortcuts can be defined by in user-configuration files, which is
-located at ``.config/mpremote/config.py``. This file should define a
-dictionary named ``commands``. The keys of this dictionary are the shortcuts
+located at ``.config/mpremote/config.py`` relative to the ``XDG_CONFIG_HOME`` or ``HOME`` environment variable on unix systems
+, or on Windows relative to ``HOME``, ``USERPROFILE`` or ``APPDATA``.
```

Comment:
That's not what the code seems to do: it tries all of "XDG_CONFIG_HOME", "HOME", "USERPROFILE", "APPDATA" irregardless of platforms?

Reply:
Yes this is just a detail. I meant this sentence makes it sound as if on unix the mpremote looks for HOME etc and on windows for USERPROFILE while in reality the code scans all variables both on unix and windows. Which is good enough in my opinion, and can be stated like 'Additional shortcuts can be defined by in the user configuration file, which is located at .config/mpremote/config.py relative to the XDG_CONFIG_HOME, HOME OR USERPROFILE environment variable (searched in that order).'